crates/viewer/re_component_ui/src/lib.rs

```diff
@@ -83,9 +84,11 @@ pub fn create_component_ui_registry() -> re_viewer_context::ComponentUiRegistry
     registry.add_singleline_edit_or_view::<GammaCorrection>(edit_f32_zero_to_max);
     registry.add_singleline_edit_or_view::<GridSpacing>(edit_f32_zero_to_max);
     registry.add_singleline_edit_or_view::<ImagePlaneDistance>(edit_f32_zero_to_max);
+    registry.add_singleline_edit_or_view::<LinearSpeed>(edit_f64_zero_to_max);
     registry.add_singleline_edit_or_view::<MarkerSize>(edit_ui_points);
-    registry.add_singleline_edit_or_view::<StrokeWidth>(edit_ui_points);
     registry.add_singleline_edit_or_view::<NearClipPlane>(edit_f32_zero_to_max);
+    registry.add_singleline_edit_or_view::<Scalar>(edit_f64_zero_to_max);
```

Comment:
let's not add `Scalar` here - I'm not sure if it might show up in places where we don't want it, no reason to risk that